source/backend/dal/Repositories/AcquisitionFileRepository.cs

```diff
@@ -62,6 +64,122 @@ public Paged<PimsAcquisitionFile> GetPage(AcquisitionFilter filter, HashSet<shor
             return new Paged<PimsAcquisitionFile>(items, filter.Page, filter.Quantity, query.Count());
         }
 
+        /// <summary>
+        /// Get an Acquisition Files Export based on Filter.
+        /// </summary>
+        /// <param name="filter"></param>
+        /// <param name="regions"></param>
+        /// <param name="filterPersonId"></param>
+        /// <returns>List of AcquisitionFileExportDto.</returns>
+        public List<AcquisitionFileExportDto> GetAcquisitionFileExport(AcquisitionFilter filter, HashSet<short> regions, long? filterPersonId = null)
```

Comment:
This seems dangerous, as we now have two filters, one that returns data for the acquisition file list view, and this one that returns data for the export. there is no guarantee that the two will return the same data, or remain in sync in the case of future updates. 

Can you explain why we need this function instead of just re-using the existing acquisition search that populates the list view? even if we don't reuse that entire repository method could we at least extract out the database query part to make sure they are the same?

Reply:
I agree that we should try to use the existing "GetPage" functionality of acquisition list view. If we wanted to get all records instead of pages we could send `PageSize = Int.MaxValue` (or whatever C# constant is for max integer). 

I don't see the need for a separate function and as Devin mentions I would be concerned about the two functions not returning the same data